### DISCUSSION TITLE
Having a problem with NTS and dns

### DISCUSSION THREAD

**Discussion by ghost** [DNSCrypt/dnscrypt-proxy#2177] [Q&A]
The LATEST version of `dnscrypt-proxy` (precompiled binaries downloaded from this repository) is correctly installed and configured on your system, but something doesn't seem to produce the expected result.

```sh
./dnscrypt-proxy -version
2.1.1

./dnscrypt-proxy -check

./dnscrypt-proxy -resolve example.com
Resolver      : 193.27.13.82
Lying         : no
DNSSEC        : yes, the resolver supports DNSSEC
```

I used the guide for NTP server with NTS support using this guide: https://fedoramagazine.org/secure-ntp-with-nts/, but when I reboot I need to restart both `dnscrypt-proxy` and `chronyd` because it seems to not resolve the names.

I don't have internet until I restart the two services when I reboot.

It happens probably in the initialization of the dns service.

I don't know if it is really a bug but it seems to be related to the dns.

**Comment by lifenjoiner** [comment]
Try to find some system logs about those 2 services, especially about `dnscrypt-proxy` ...

**Reply by ghost** [reply]
Sorry, I forgot to come back here! So, it seems that the mullvad dns was having problems, I changed to the quad9 and it solved my problem.
